### PR TITLE
Internet Identity How-To pages

### PR DESCRIPTION
**Pull request by michelabdalla-dfinity**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: how-it-works/6_authentication/01_web-authentication-identity.card.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 ---
 title: Internet Identity
-abstract: The Internet Computer blockchain replaced usernames and passwords with a more advanced and secure method of cryptographic authentication.
+abstract:  The Internet Identity is an advanced and secure method of cryptographic authentication, which is convenient to use, works across all of a user's devices, and helps to protect user privacy.
```
Comment (Dfinity-Bjoern) [inline how-it-works/6_authentication/01_web-authentication-identity.card.md:3]:
```suggestion
abstract:  Internet Identity is an advanced and secure method of cryptographic authentication, which is convenient to use, works across all of a user's devices, and helps to protect user privacy.
```

File: how-it-works/6_authentication/01_web-authentication-identity.card.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 coverImage: /img/how-it-works/web-authentication-identity.600x300.jpg
 ---
 
 ![](/img/how-it-works/web-authentication-identity.600x300.jpg)
 
 # Internet Identity
 
-Today, the main means of identity and authentication prevalently used online are usernames and passwords. The Internet Computer blockchain has replaced this model with a more advanced and much more secure method of cryptographic authentication that is more convenient, works across all of a user's devices, and helps to protect user privacy.
+The main means of identity and authentication used on the web are usernames and passwords, which are hard to manage and well-known for their security vulnerabilities. To solve these pitfalls, the Internet Computer blockchain pioneered a more advanced and much more secure method of cryptographic authentication, known as the Internet Identity, which is more convenient to use, works across all of a user's devices, and helps to protect user privacy.
```
Comment (Dfinity-Bjoern) [inline how-it-works/6_authentication/01_web-authentication-identity.card.md:11]:
```suggestion
The main means of identity and authentication used on the web are usernames and passwords, which are hard to manage and well-known for their security vulnerabilities. To solve these pitfalls, the Internet Computer blockchain pioneered a more advanced and much more secure method of cryptographic authentication, known as Internet Identity, which is more convenient to use, works across all of a user's devices, and helps to protect user privacy.
```

File: how-it-works/6_authentication/01_web-authentication-identity.subpage.md [hunk 3]
```diff
@@ -7,16 +7,31 @@ slug: web-authentication-identity
 
 # Internet Identity
 
-[![Watch youtube video](https://i.ytimg.com/vi/9eUTcCP_ELM/maxresdefault.jpg)](https://www.youtube.com/watch?v=9eUTcCP_ELM)
+In order to access and interact with applications running on the Internet Computer, it is often necessary for users to authenticate. One of the most commonly used methods to authenticate on the Internet Computer is via the Internet Identity.
```
Comment (Dfinity-Bjoern) [inline how-it-works/6_authentication/01_web-authentication-identity.subpage.md:10]:
```suggestion
In order to access and interact with applications running on the Internet Computer, it is often necessary for users to authenticate. One of the most commonly used methods to authenticate on the Internet Computer is Internet Identity.
```

File: how-it-works/6_authentication/01_web-authentication-identity.subpage.md [hunk 4]
```diff
@@ -7,16 +7,31 @@ slug: web-authentication-identity
...
+
+The [Internet Identity](https://identity.ic0.app/) is a blockchain authentication framework supported by the Internet Computer. Users begin the process by creating identity "anchors" to which they assign compatible cryptographically enabled devices, such as the fingerprint sensor on a laptop, the face ID system on a phone, or a portable HSM, such as a YubiKey or Ledger wallet. Thereafter, they can signup and authenticate to dapps running on the Internet Computer using any of the devices they have assigned to their anchor.
```
Comment (Dfinity-Bjoern) [inline how-it-works/6_authentication/01_web-authentication-identity.subpage.md:12]:
```suggestion
[Internet Identity](https://identity.ic0.app/) is a blockchain authentication framework supported by the Internet Computer. Users begin the process by creating identity "anchors" to which they assign compatible cryptographically enabled devices, such as the fingerprint sensor on a laptop, the face ID system on a phone, or a portable HSM, such as a YubiKey or Ledger wallet. Thereafter, they can signup and authenticate to dapps running on the Internet Computer using any of the devices they have assigned to their anchor.
```

**Review by Dfinity-Bjoern — APPROVED** [review]
LGTM doing some changes

**Review by aisconnolly — APPROVED** [review]

**Pull request merged**